src/components/WeatherWidget.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="weather-widget">
+    <WeatherWidgetPlace
+      :="weatherData.location"
+      :date="selectedDayData.date ?? 'Today'"
```

Comment:
this is interesting way of binding props, I don't know if I'd like to see that in a real project, but maybe I'm not used to 

let's wait for other guys opinions

Reply:
Resolved.